src/SecureSign.Web/Controllers/AuthenticodeSigningController.cs

```diff
@@ -50,14 +50,14 @@ public async Task<IActionResult> SignUsingAuthenticode(SignRequest request)
 			}
 
 			var cert = _secretStorage.LoadAuthenticodeCertificate(token.KeyName, token.Code);
-			var (artifact, artifactError) = await _utils.GetFileFromPayloadAsync(token, tokenConfig, request);
+			var (artifact, artifactError, fileExtention) = await _utils.GetFileFromPayloadAsync(token, tokenConfig, request);
 			if (artifactError != null)
 			{
 				return artifactError;
 			}
-
-			var signed = await _signer.SignAsync(artifact, cert, tokenConfig.SignDescription, tokenConfig.SignUrl);
-			return File(signed, "application/octet-stream");
+            
+            var signed = await _signer.SignAsync(artifact, cert, tokenConfig.SignDescription, tokenConfig.SignUrl, fileExtention);
+            return File(signed, "application/octet-stream");
```

Comment:
Please use tabs instead of spaces for indentation.

I normally put an `.editorconfig` in the repo so Visual Studio automatically uses the right settings, but I guess I forgot to do that in this repo.

Reply:
Not a problem, yeah I am set for spaces by default.